SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -648,11 +648,6 @@ void CIncNSSolver::SetTau_Wall_WF(CGeometry *geometry, CSolver **solver_containe
       const auto iPoint = geometry->vertex[iMarker][iVertex]->GetNode();
       const auto Point_Normal = geometry->vertex[iMarker][iVertex]->GetNormal_Neighbor();
 
-      /*--- Check if the node belongs to the domain (i.e, not a halo node)
-       *    and the neighbor is not part of the physical boundary ---*/
-
-      if (!geometry->nodes->GetDomain(iPoint)) continue;
```

Comment:
What about the full MG thing?

Reply:
I've made an update now that correctly runs for MG, but I do not have a working testcase for FULLMG_CYCLE. If I just switch on full mg for the turbulent incompressible flat plate, it gives me a bunch of inf values. 
Is there a working test case somewhere, or a testcase known to work when switching to FULLMG?

Reply:
When I switch to a laminar flat plate and run in serial, the FULLMG_CYCLE still fails (FGMRES orthogonalization failed, linear solver diverged). So I guess there are other options that I have to change as well, e.g. in the linear solver?

Reply:
```
+------------------------------------------------------------------------------------------+
|  Inner_Iter|      rms[P]|      rms[U]|      rms[k]|      rms[w]|          CL|          CD|
+------------------------------------------------------------------------------------------+
|           0|        -inf|        -inf|        -inf|        -inf|    0.000000|    0.000000|
|           1|        -inf|        -inf|        -inf|        -inf|    0.000000|    0.000000|
|           2|        -inf|        -inf|        -inf|        -inf|    0.000000|    0.000000|


```

Reply:
I don't have a case that uses it. I think it is better to not waste time with it